### ISSUE TITLE
Typescript

### ISSUE THREAD

**Issue by schmidtsonian** [cinkonaap/pixi-dragonbones#2]
Is there any typescrypt source of this plugin?

**Comment by cinkonaap** [comment]
Nope. I don't have much time to take proper care of this library right now, so even JS sources ain't finished.

If I had time to do it, I would go with ES6 anyway.
